spectacles/lookml.py

```diff
@@ -429,6 +433,10 @@ def build_project(
         if include_dimensions:
             for explore in model.explores:
                 explore.dimensions = build_dimensions(client, model.name, explore.name)
+                if ignore_hidden_fields:
+                    explore.dimensions = [
+                        dim for dim in explore.dimensions if not dim.is_hidden
+                    ]
```

Comment:
Should we move this logic into `build_dimensions`? That's already where we filter out `ignore` so it's probably more consistent.